diff --git a/node_modules/werift/lib/ice/src/ice.js b/node_modules/werift/lib/ice/src/ice.js
index eff8509..9e15c4e 100644
--- a/node_modules/werift/lib/ice/src/ice.js
+++ b/node_modules/werift/lib/ice/src/ice.js
@@ -436,7 +436,7 @@ class Connection {
         };
         const { stunServer, turnServer } = this.options;
         this.stunServer = (0, iceBase_1.validateAddress)(stunServer) ?? [
-            "stun.l.google.com",
+            "",
             19302,
         ];
         this.turnServer = (0, iceBase_1.validateAddress)(turnServer);
diff --git a/node_modules/werift/lib/ice/src/iceBase.js b/node_modules/werift/lib/ice/src/iceBase.js
index 400769c..e19b639 100644
--- a/node_modules/werift/lib/ice/src/iceBase.js
+++ b/node_modules/werift/lib/ice/src/iceBase.js
@@ -146,7 +146,8 @@ async function serverReflexiveCandidate(protocol, stunServer) {
         if (!localCandidate) {
             throw new Error("not exist");
         }
-        const candidate = new candidate_1.Candidate((0, candidate_1.candidateFoundation)("srflx", "udp", localCandidate.host), localCandidate.component, localCandidate.transport, (0, candidate_1.candidatePriority)("srflx"), response.getAttributeValue("XOR-MAPPED-ADDRESS")[0], response.getAttributeValue("XOR-MAPPED-ADDRESS")[1], "srflx", localCandidate.host, localCandidate.port);
+        const mappedAddress = response.getAttributeValue("XOR-MAPPED-ADDRESS") ?? response.getAttributeValue("MAPPED-ADDRESS")
+        const candidate = new candidate_1.Candidate((0, candidate_1.candidateFoundation)("srflx", "udp", localCandidate.host), localCandidate.component, localCandidate.transport, (0, candidate_1.candidatePriority)("srflx"), mappedAddress[0], mappedAddress[1], "srflx", localCandidate.host, localCandidate.port);
         return candidate;
     }
     catch (error) {
diff --git a/node_modules/werift/lib/ice/src/utils.js b/node_modules/werift/lib/ice/src/utils.js
index da83909..c26fd64 100644
--- a/node_modules/werift/lib/ice/src/utils.js
+++ b/node_modules/werift/lib/ice/src/utils.js
@@ -16,7 +16,7 @@ async function getGlobalIp(stunServer, interfaceAddresses) {
     const protocol = new protocol_1.StunProtocol();
     await protocol.connectionMade(true, undefined, interfaceAddresses);
     const request = new message_1.Message(const_1.methods.BINDING, const_1.classes.REQUEST);
-    const [response] = await protocol.request(request, stunServer ?? ["stun.l.google.com", 19302]);
+    const [response] = await protocol.request(request, stunServer ?? ["", 19302]);
     await protocol.close();
     const address = response.getAttributeValue("XOR-MAPPED-ADDRESS");
     return address[0];
diff --git a/node_modules/werift/lib/index.mjs b/node_modules/werift/lib/index.mjs
index 3243466..988dfc1 100644
--- a/node_modules/werift/lib/index.mjs
+++ b/node_modules/werift/lib/index.mjs
@@ -7540,7 +7540,7 @@ async function getGlobalIp(stunServer, interfaceAddresses) {
   const request = new Message(1 /* BINDING */, 0 /* REQUEST */);
   const [response] = await protocol.request(
     request,
-    stunServer ?? ["stun.l.google.com", 19302]
+    stunServer ?? ["", 19302]
   );
   await protocol.close();
   const address = response.getAttributeValue("XOR-MAPPED-ADDRESS");
@@ -7596,7 +7596,7 @@ var Connection = class {
     };
     const { stunServer, turnServer } = this.options;
     this.stunServer = validateAddress(stunServer) ?? [
-      "stun.l.google.com",
+      "",
       19302
     ];
     this.turnServer = validateAddress(turnServer);
@@ -14635,7 +14635,7 @@ var defaultPeerConfig = {
     video: []
   },
   iceTransportPolicy: "all",
-  iceServers: [{ urls: "stun:stun.l.google.com:19302" }],
+  iceServers: [{ urls: "" }],
   icePortRange: void 0,
   iceInterfaceAddresses: void 0,
   iceAdditionalHostAddresses: void 0,
diff --git a/node_modules/werift/lib/nonstandard/index.mjs b/node_modules/werift/lib/nonstandard/index.mjs
index 38adf29..04e4ed1 100644
--- a/node_modules/werift/lib/nonstandard/index.mjs
+++ b/node_modules/werift/lib/nonstandard/index.mjs
@@ -9207,7 +9207,7 @@ var defaultPeerConfig = {
     video: []
   },
   iceTransportPolicy: "all",
-  iceServers: [{ urls: "stun:stun.l.google.com:19302" }],
+  iceServers: [{ urls: "" }],
   icePortRange: void 0,
   iceInterfaceAddresses: void 0,
   iceAdditionalHostAddresses: void 0,
diff --git a/node_modules/werift/lib/webrtc/src/peerConnection.js b/node_modules/werift/lib/webrtc/src/peerConnection.js
index dbd96bf..610637b 100644
--- a/node_modules/werift/lib/webrtc/src/peerConnection.js
+++ b/node_modules/werift/lib/webrtc/src/peerConnection.js
@@ -1500,7 +1500,7 @@ exports.defaultPeerConfig = {
         video: [],
     },
     iceTransportPolicy: "all",
-    iceServers: [{ urls: "stun:stun.l.google.com:19302" }],
+    iceServers: [{ urls: "" }],
     icePortRange: undefined,
     iceInterfaceAddresses: undefined,
     iceAdditionalHostAddresses: undefined,
